addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd

```diff
@@ -39,6 +39,6 @@ func run(command_params: Array) -> int:
 		.push(
 			escoria.object_manager.get_object(command_params[0]).node,
 			command_params[1],
-			command_params[2]
+			Tween.new().get("TRANS_%s" % command_params[2])
```

Comment:
If I understand correctly, the idea here is to get the right enum value from Tween, using command_params[2]. Does this really require creating a Tween object?

Reply:
I used that code like it was before. Is there another way to get the Enum value from a string?

Reply:
I thought there was... After a search, it seems there is not. Ok for me then.